frontend/src/components/Rating.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+const Rating = ({ value, numReviews, color }) => {
+
+    return (
+        <div className="rating">
+            <span>
```

Comment:
Code is getting repeated here, please check if we can use a loop to map those span elements

Reply:
done